docs/_documentations/codewind-vs-code-quick-guide.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: docs
+title: Codewind in VS Code
+description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
+keywords: Codewind, VS Code, microservice
+duration: 5 minutes
+permalink: codewind-vscode-quick-guide
+---
+
+# Getting Started with Codewind in VS Code
+
+## Objectives
+* Install Visual Studio Code (VS Code) and Codewind 
+* Develop a simple microservice, using Eclipse Codewind on VS Code
+
+## Overview 
+Eclipse Codewind provides the ability to create application projects from `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes. Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.    
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters, OpenShift Do (odo), and Appsody templates. Today, there are templates for: IBM Cloud Starters, odo, Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, and Node.js with Loopback.
+
+## Developing with VS Code
+You can use Codewind for VS Code to develop and debug your containerized projects from within VS Code using the workflow you already use today.
+
+### Prerequisite 
+Before you can develop a microservice with VS Code, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install VS Code](https://code.visualstudio.com/download)
+ 
+### Installing Codewind for VS Code
```

Comment:
Per my conversation with Naveen, perhaps we can include a note about the pop window that appears regarding Docker images? Might not be necessary for a quick guide's purposes. See this image: 

![](https://files.slack.com/files-pri/T27SFGS2W-F0136ND0FN1/image.png)